src/Assets/scss/_form-elements.scss

```diff
@@ -49,6 +49,10 @@
         min-height: 50px;
         border-color: $secondary-color;
         background: none;
+        &.invalid{
+          border: 1px solid #dc3545;
```

Comment:
@maheshkurma Instead of adding #dc3545 as an inline style add in _variable.scss and use that defined variable in _form-elements.scss

Reply:
updated.

Reply:
+1